com/alex/PersonManager.java

```diff
@@ -14,7 +14,8 @@
      */
     public Person updateFirstName(int id, String firstName) {
         Person person = findPeron(id);
-        person.setFirstName(firstName);
+        String newFirstName = firstName;
+        person.setFirstName(newFirstName);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Potential violation of Law of Demeter (object not created locally)](https://www.codacy.com/app/alex-ananighian/CodacyTest/file/2736606337/issues/source?bid=3351995&fileBranchId=3352254#l18)